core/src/test/java/com/linecorp/armeria/server/AnnotatedHttpServiceTest.java

```diff
@@ -423,7 +474,7 @@ public void testAnnotatedHttpService() throws Exception {
             }
             // Run case 3 but with not-mapped HTTP method (Post).
             try (CloseableHttpResponse res = hc.execute(new HttpPost(rule.httpUri("/1/string/blah")))) {
-                assertThat(res.getStatusLine().toString(), is("HTTP/1.1 405 Method Not Allowed"));
+                assertThat(res.getStatusLine().toString(), is("HTTP/1.1 404 Not Found"));
```

Comment:
I'm curious if we should still return 405 because there's a mapping for this path and it's just a certain method that's unsupported.